### PR TITLE
PS-7090 Update documentation to reflect 8.0.20 parallel doublewrite c…

### PR DESCRIPTION
**Pull request by patrickbirch**
…hanges

Modified changed_in_version
Modified performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst
Modified scalability/innodb_io.rst
Modified security/encrypting-doublewrite-buffers.rst

### CODE REVIEW

**Review by kathy-williamson — APPROVED** [review]

**Review by george-lorch — APPROVED** [review]

File: doc/source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst [hunk 1]
```diff
@@ -141,6 +89,8 @@ setting ``O_DIRECT`` has failed.
    :vartype: String
    :default: ``xb_doublewrite``
 
+As of |Percona Server| 8.0.20-11, this variable is considered **deprecated** and is no longer configurable. You should use `innodb_doublewrite_dir <https://dev.mysql.com/doc/refman/8.0/en/innodb-parameters.html#sysvar_innodb_doublewrite_dir>`_.
```
Comment (percona-ysorokin) [inline doc/source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst]:
```suggestion
As of |Percona Server| 8.0.20-11, this variable is considered **deprecated** and has no effect. You should use `innodb_doublewrite_dir <https://dev.mysql.com/doc/refman/8.0/en/innodb-parameters.html#sysvar_innodb_doublewrite_dir>`_.

File: doc/source/scalability/innodb_io.rst [hunk 2]
```diff
@@ -27,25 +27,26 @@ System Variables
...
-       the ``fsync()`` system call to flush the parallel doublewrite file.
+     - Use ``O_SYNC`` to open and flush doublewrite files; Do not use
+       the ``fsync()`` system call to flush the doublewrite file.
    * - O_DIRECT
      - Use O_DIRECT to open the data files and the ``fsync()`` system call to flush
-       parallel doublewrite files.
+       doublewrite files.
    * - O_DIRECT_NO_FSYNC
      - Use ``O_DIRECT`` to open the data files but don't use ``fsync()`` system
        call to flush doublewrite files.
```
Comment (percona-ysorokin) [inline doc/source/scalability/innodb_io.rst]: This table is no longer accurate and should be completely removed.

The statement above with `affects doublewrite buffers exactly the same as in |MySQL| 8.0.20` should suffice.

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

File: doc/source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst]: Updated.

File: doc/source/scalability/innodb_io.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/scalability/innodb_io.rst]: Removed table

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**